app/javascript/dashboard/components/layout/sidebarComponents/NotificationBell.vue

```diff
@@ -1,19 +1,32 @@
 <template>
   <div class="notifications-link">
-    <primary-nav-item
-      name="NOTIFICATIONS"
-      icon="alert"
-      :to="`/app/accounts/${accountId}/notifications`"
-      :count="unreadCount"
+    <button
```

Comment:
Can we use woot-button here? It handles states like active, hover etc

Reply:
fixes by https://github.com/chatwoot/chatwoot/pull/4061/commits/8ae236b568fa2cd316395131f5222bb81f5743e6